Modix.Services/Core/AuthorizationService.cs

```diff
@@ -238,12 +238,14 @@ public void RequireAuthenticatedUser()
         {
             if (CurrentUserId == null)
                 // TODO: Booooo for exception-based flow control
-                throw new InvalidOperationException("The current operation requires an authenticated guild.");
+                throw new InvalidOperationException("The current operation requires an authenticated user.");
         }
 
         /// <inheritdoc />
         public void RequireClaims(params AuthorizationClaim[] claims)
         {
+            RequireAuthenticatedUser();
```

Comment:
I mean, I guess there's no scenario where we could ever load claims without a user, but..... I don't really see the need for this as a requirement. /shrug

Reply:
Just a consistency thing, mostly. Without it you get a mysterious "required parameter 'second'" exception on the call to `Except()` if you haven't already called OnAuthenticatedAsync.

Reply:
...wut?

Reply:
I know! It was very confusing:
![image](https://user-images.githubusercontent.com/1239222/43988754-179d0122-9d0a-11e8-9b12-77c47487be05.png)